mewarpx/mewarpx/mwxconstants.py

```diff
@@ -1,11 +1,24 @@
 """File to keep constants not already in picmi.py that are needed in mewarpx."""
 from pywarpx.picmi import constants
 
+import numpy as np
+import scipy.constants as scipy_consts
+
 # CONSTANTS - SI
+physical_constants_dict = scipy_consts.codata._physical_constants_2018
+h = physical_constants_dict['Planck constant'][0]
+e = physical_constants_dict['elementary charge'][0]
+m_e = physical_constants_dict['electron mass'][0]
```

Comment:
No, we want to be using PICMI constants by default. picmi.constants already defines all these things. Feel free to copy them here, eg
```
e = constants.e
h = constants.hbar * 2*np.pi
```
The reason is that Picmi mirrors exactly what's used in WarpX, so we want to stick with that too.